### ISSUE TITLE
Improve RefParser package

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1223]
> @JJK96 just thought you might want to check these forks of the reference parser:
> https://github.com/renehamburger/Bible-Passage-Reference-Parser (improvements in German and other repo maintenance improvements)
> https://github.com/alerque/Bible-Passage-Reference-Parser (improvements in Turkish)

I can include improved versions in RefParser package

_Originally posted by @tuomas2 in https://github.com/AndBible/and-bible/issues/981#issuecomment-850872782_

**Comment by JJK96** [comment]
We could maintain our own fork, but I prefer just sticking with the original for now. None of us use German or Turkish, let's just see what other users are thinking. I think it's a bit early to start using other forks to fix some edge cases that nobody has ran into yet

**Comment by tuomas2** [comment]
I agree it's by no means high prio and we need to evaluate first how important those improvements are.
